test-poc/framework/src/main/java/org/keycloak/test/framework/server/KeycloakTestServerSupplier.java

```diff
@@ -23,17 +23,12 @@ public Class<KeycloakIntegrationTest> getAnnotationClass() {
     public InstanceWrapper<KeycloakTestServer, KeycloakIntegrationTest> getValue(Registry registry, KeycloakIntegrationTest annotation) {
         KeycloakTestServerConfig serverConfig = SupplierHelpers.getInstance(annotation.config());
 
-//        RemoteKeycloakTestServer keycloakTestServer = new RemoteKeycloakTestServer();
-        EmbeddedKeycloakTestServer keycloakTestServer = new EmbeddedKeycloakTestServer();
+        RemoteKeycloakTestServer keycloakTestServer = new RemoteKeycloakTestServer();
+//        EmbeddedKeycloakTestServer keycloakTestServer = new EmbeddedKeycloakTestServer();
```

Comment:
This is already present as a line of code above, so perhaps this comment can be removed?
```suggestion
```

Reply:
Wops, didn't mean to commit this change - just changed it to remote temporarily while playing around with things. Reverted now.